legateboost/legateboost.py

```diff
@@ -54,6 +54,29 @@ def __init__(
             raise ValueError("base_models must be a tuple")
         self.base_models = base_models
 
+        # define what happens to the attributes when two models are added
+        self._add_behaviour.update(
+            {
+                "models_": AddMember.ADD,
+                "model_init_": AddMember.ADD,
+                "n_features_in_": AddMember.ASSERT_SAME,
+                "is_fitted_": AddMember.ASSERT_SAME,
+                "n_estimators": AddMember.PREFER_A,
```

Comment:
shouldn't this be `ADD`?

Reply:
Yes I think it makes sense to be ADD here.